foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -149,7 +149,7 @@ The parent, child, and sibling relationships between elements will become much m
 
 HTML comments are not visible to the browser; they allow us to *comment* on our code so that other developers or our future selves can read them and get some context about something that might not be clear in the code.
 
-Writing an HTML comment is simple: We just enclose the comment with `<!--` and `-->` tags. For example:
+Writing an HTML comment: We just enclose the comment with `<!--` and `-->` tags. For example:
```

Comment:
I think removing the "is simple" makes this sound a little clunky. Instead maybe:

```suggestion
In order to write an HTML comment, we just enclose the comment with `<!--` and `-->` tags. For example:
```